frontend/pages/sign-up.tsx

```diff
@@ -0,0 +1,299 @@
+            )}
+          </div>
+        </div>
+        <div className="w-full">
+          <label htmlFor="email">
+            <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+              <FormattedMessage defaultMessage="Email" id="sy+pv5" />
+            </p>
+            <Input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Insert your email"
+              aria-describedby="email-error"
+              register={register}
+              registerOptions={{
+                required: intl.formatMessage({
+                  defaultMessage: 'You need to enter your email',
+                  id: 'NioTuC',
+                }),
+                validate: (value) =>
+                  validate(
+                    { from: value },
+                    {
+                      from: {
+                        email: true,
+                      },
+                    }
+                  ),
+              }}
+            />
+          </label>
+          {errors.email && (
+            <p id="email-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+              {errors.email.message || (
+                <FormattedMessage defaultMessage="Invalid email format" id="nc1IrM" />
+              )}
+            </p>
+          )}
+        </div>
+        <div className="flex gap-4">
+          <div className="w-full">
+            <label htmlFor="password">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Password" id="5sg7KC" />
+              </p>
+              <Input
+                type="password"
+                placeholder="insert password"
+                name="password"
+                id="password"
+                aria-describedby="password-description password-error"
+                register={register}
+                registerOptions={{
+                  required: intl.formatMessage({
+                    defaultMessage: 'You need to enter a password.',
+                    id: 'zeCjLr',
+                  }),
+                  minLength: {
+                    value: 8,
+                    message: intl.formatMessage({
+                      defaultMessage: 'The password must have at least 8 characters.',
+                      id: 'TiXJ+4',
+                    }),
+                  },
+                }}
+              />
+            </label>
+            <p
+              className="mt-1 font-sans text-xs text-gray-400 font-regular"
+              id="password-description"
+            >
+              <FormattedMessage defaultMessage="Use at least 8 characters." id="BvrO01" />
+            </p>
+            {errors.password && (
+              <p id="password-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+                {errors.password.message}
+              </p>
+            )}
```

Comment:
I know this is how it is on the UI kit, but I think it looks weird that the help text and error are not aligned. Do you mind making a change and/or review with Andreia?

![Help text and error are not aligned horizontally](https://user-images.githubusercontent.com/6073968/159653267-e9b02863-62ae-467f-9d02-bf9602e19a85.png)